resources/domain/request/index.ts

```diff
@@ -7,10 +13,16 @@ export const isCancelableRequest = (status: string): Boolean => {
 
 export const getSignatureRequestClass = ({ status = '' }) => `signerRequest ${capitalize(status)}`
 
-export const isSignatureRequest = (requestType: string): requestType is SignatureRequestType => {
+export const isSignatureRequestType = (requestType: string): requestType is SignatureRequestType => {
```

Comment:
kind of like the previous naming tbh, I think its good practice to think of objects in our code as they actually _are_, not as the abstract types that represent them